atd-events/crash_update_jurisdiction/app.py

```diff
@@ -67,6 +69,46 @@ def get_jurisdiction_flag(data: dict) -> str:
         return "N"
 
 
+def get_city_id(data: dict) -> Optional[int]:
```

Comment:
Cool annotations here! I haven't seen this syntax before so it was fun looking into it. Is there any type checking here or is it for documentation? I read conflicting things so I thought that I would just ask. 👍

Reply:
There is no type checking, only syntax. The only reason of these annotations is to help the developers understand what they are working with.

Reply:
Awesome, thanks! They are very helpful. 👍